backend/cypress/e2e/createNewAudit.cy.js

```diff
@@ -37,6 +37,6 @@ describe('Create new audit', () => {
   });
 
   it('Fills out the General Information Form', () => {
-    testValidGeneralInfo();
+    GeneralInfoForm();
   });
-});
+});
```

Comment:
I'd put this newline back so GH stops complaining. Some of your other files are missing EoF newlines as well. [Here's](https://stackoverflow.com/a/729795) why they're good.